Kros.KORM/src/Kros.KORM.Extensions.Asp/ServiceCollectionExtensions.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using Kros.Extensions;
+using Kros.KORM.Extensions.Asp.Properties;
```

Comment:
Is this using necessary?

Reply:
Yes, because I moved `Resources` into `Properties` namespace.